Runtime/Core/ApiGatewayManagement/AnywhereGameServerAdapter.cs

```diff
@@ -0,0 +1,119 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Amazon.GameLift;
+using AmazonGameLiftPlugin.Core.ApiGatewayManagement.Models;
+using AmazonGameLiftPlugin.Core.Shared;
+using UnityEngine;
+using Logger = AmazonGameLiftPlugin.Core.Shared.Logging.Logger;
```

Comment:
Are we using a different `Logger` from before. Asking since I see this being directly referenced without any imports in the `LocaltGameAdapter.cs`, so trying to understand what the difference is here. Thanks!

Reply:
Yep was a mistake, UnityEngine was confusing the logger. I have solved this now.